.github/workflows/e2e.yml

```diff
@@ -1,3 +1,4 @@
+# If you change this name also do it in ci_metrics.yml
 name: end-to-end
```

Comment:
Absolute nitpick, now I'm triggered about the fact that every workflow uses "Nice Spacing and Capital Letters" while this is "lowercase-and-dashes-all-the-way" :smile: Time to rename it? :grin:

Reply:
It's not top priority but am planning a PR to introduce some naming standards and rename the existing ones. 🧑‍💻